pkg/archive/archive.go

```diff
@@ -151,6 +154,11 @@ func DetectCompression(source []byte) Compression {
 	return Uncompressed
 }
 
+func zstdDecompress(ctx context.Context, archive io.Reader) (io.ReadCloser, error) {
```

Comment:
These are probably legacy forms, I'll investigate.

re: zstd extension, this would be for `ADD foo.zst`. We could add it, though it's not really my intention here.

Reply:
Ah, yes, forgot that the other one also implemented _exporting_ / _saving_ (and for that used  `.tar.zst` file-extension for the exported file)